--- hosts	2019-05-05 10:02:16.769986218 +0000
+++ hosts.ok	2019-05-05 09:59:41.389986218 +0000
@@ -14,14 +14,16 @@
 #blue.example.com
 #192.168.100.1
 #192.168.100.10
+acs ansible_connection=local
 
 # Ex 2: A collection of hosts belonging to the 'webservers' group
 
-#[webservers]
+[webservers]
 #alpha.example.org
 #beta.example.org
 #192.168.1.100
 #192.168.1.110
+web ansible_host=192.168.33.20
 
 # If you have multiple hosts following a pattern you can specify
 # them like this:
@@ -30,12 +32,13 @@
 
 # Ex 3: A collection of database servers in the 'dbservers' group
 
-#[dbservers]
+[dbservers]
 #
 #db01.intranet.mydomain.net
 #db02.intranet.mydomain.net
 #10.25.1.56
 #10.25.1.57
+db ansible_host=192.168.33.30
 
 # Here's another example of host ranges, this time there are no
 # leading 0s:
